integration_tests/suite/test_config.py

```diff
@@ -66,3 +71,32 @@ def _not_return_503():
                 raise AssertionError(e)
 
         until.assert_(_not_return_503, tries=10)
+
+    def test_update_config(self) -> None:
+        debug_true_config = [
+            {
+                'op': 'replace',
+                'path': '/debug',
+                'value': True,
+            }
+        ]
+
+        debug_false_config = [
+            {
+                'op': 'replace',
+                'path': '/debug',
+                'value': False,
+            }
+        ]
+
+        dird = self.make_dird(VALID_TOKEN_MAIN_TENANT)
+
+        debug_true_patched_config = dird.config.patch(debug_true_config)
+        debug_true_config = dird.config.get()
```

Comment:
You should not reuse the variable name `debug_true_config` here. It make the test confusing.
At the begining of the test the value is
```python
[
            {
                'op': 'replace',
                'path': '/debug',
                'value': True,
            }
        ]
```
and at the end it's
```python
{
    ...
     "debug": True,
     ...
 }
 ```
 You could name it `debug_true_updated_config`

Reply:
Same comment for `debug_false_config`

Reply:
Thank you, It is done.